.gitignore

```diff
@@ -1,2 +1,3 @@
 git node_modules
```

Comment:
- [ ] Good job, but this one has no impact, and it's highly recommended to remove the unnecessary lines of code from your files. It'll help you keep your repository cleaner. Kindly make sure to remove it.

Reply:
Thanks for highlighting.